clar2wasm/src/bin/utils/mod.rs

```diff
@@ -0,0 +1,82 @@
+use clap::builder::PossibleValue;
+use clap::ValueEnum;
+use clarity::types::StacksEpochId;
+use clarity::vm::ClarityVersion;
+
+#[derive(Clone)]
+pub struct WrappedEpochId(StacksEpochId);
+
+impl Default for WrappedEpochId {
+    fn default() -> WrappedEpochId {
+        WrappedEpochId(StacksEpochId::latest())
```

Comment:
Maybe it's because I come from the embedded world, but I'm not sure using `latest` automatically is wise. We could update the clarity dependency and have a different epoch without knowing, which would have bugs with the compiler.
Also, people who will use the compiler should know exactly what the epoch will be if they don't specify the argument.

Reply:
Ah, good. I was unsure about this, especially since we [don't yet support clarity 3](https://github.com/stacks-network/clarity-wasm/issues/428). I'll put clarity 2 as a default.

Reply:
Made the change